cg/apps/sequencing_metrics_parser/parsers/bcl2fastq.py

```diff
@@ -36,7 +37,8 @@ def parse_bcl2fastq_raw_tile_metrics(
 
     for stats_json_path in stats_json_paths:
         LOG.debug(f"Parsing stats.json file {stats_json_path}")
-        sequencing_metrics = Bcl2FastqSampleLaneTileMetrics.parse_file(stats_json_path)
+        data = read_json(stats_json_path)
+        sequencing_metrics = Bcl2FastqSampleLaneTileMetrics.model_validate(data)
```

Comment:
I think you don't really need to open and read the json, given that there is a model_validate_json function in pydantic v2

```suggestion
        sequencing_metrics = Bcl2FastqSampleLaneTileMetrics.model_validate_json(stats_json_path.open())
```

Reply:
Nice 👍

Reply:
Hm, it broke some tests doing it like that. I'll keep it as is since it works